presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraDistributedQueries.java

```diff
@@ -32,7 +32,7 @@
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        return CassandraQueryRunner.createCassandraQueryRunner();
+        return new CassandraQueryRunner().createCassandraQueryRunner();
```

Comment:
You want to retain ref to `CassandraServer` so that you close the docker container in `@AfterClass`

it may be simpler to write if `CassandraQueryRunner` remains a utility class and simply takes the server as a param:

```
protected QueryRunner createQueryRunner()
{ 
  this.server = new CassandraServer();
  return CassandraQueryRunner.createCassandraQueryRunner(server);
}

@AfterClass(alwaysRun=true)
public void tearDown()
{
  this.server.close();
}
```